etna/models/nn/deepar_new.py

```diff
@@ -0,0 +1,383 @@
+from collections import Counter
```

Comment:
Let's create separate folder for this model with
- Model implementation
- Sampler implementation
- Losses implementation